ui/js/component/fileCard/view.jsx

```diff
@@ -83,6 +91,7 @@ class FileCard extends React.PureComponent {
               <div className="card__title" title={title}>
                 <TruncatedText lines={1}>{title}</TruncatedText>
               </div>
+              {isHotRightNow && <span>&#128293; Hot Right Now</span>}
```

Comment:
This line appears in 3 places (file tile, file card and the Show page). If it's going to look pretty much the same in all 3 places, then this should probably factored out into a presentational component (like `<CreditAmount />` et al.)

If it's going to look the same way in 2 places and then totally different somewhere else, it's a gray area, IMO.